pkgs/development/interpreters/joker/default.nix

```diff
@@ -17,6 +17,8 @@ buildGoModule rec {
     go generate ./...
   '';
 
+  subPackages = [ "." ];
```

Comment:
@ryantm is this a new feature for `buildGoModule`? This is probably not needed if the value is `.`.

Reply:
>is this a new feature for buildGoModule?

@kalbasit sorry, I added that change because it was building unrelated packages
```
🗁  ydbf8885yk2i9qzdi6zwjfky4y9h6h55-joker-0.14.1
└── 🗁  bin
   ├── 🗋  gen
   ├── 🗋  gen_data
   ├── 🗋  joker
   └── 🗋  sum256dir
```